boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -172,14 +183,15 @@ arduino_serial: &uart1 {
 	};
 };
 
-&spi4 {
+arduino_spi: &spi4 {
 	compatible = "nordic,nrf-spim";
 	status = "okay";
-	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>,
+		<&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
 	pinctrl-0 = <&spi4_default>;
 	pinctrl-1 = <&spi4_sleep>;
 	pinctrl-names = "default", "sleep";
-	sdhc0: sdhc@0 {
+	sdhc0: sdhc@1 {
```

Comment:
Why was this changed to 1?

Reply:
It was suggested by anangl above that shield users will probably use CS0, hence sdhc and cs47l63 should be incremented by 1.